src/components/login/Login.tsx

```diff
@@ -340,7 +340,7 @@ export const Login = () => {
 				}
 
 				if (Object.keys(patchedUserData).length > 0) {
-					await apiPatchUserData(patchedUserData);
```

Comment:
Wenn hier ein Fehler kam wurde der in der function tryLogin abgefangen und der Button wurde wieder enabled.
Heißt jetzt nicht, dass es deshalb funktioniert wenn er nochmal auf den Button klickt aber jetzt wird er in die App geleitet obwohl etwas schief glaufen ist.
Weiß jetzt auch nicht welches verhalten besser ist ehrlich gesagt. :-D

Reply:
Ist okay so. Wenn das patchUser fehlschlägt ist nix kritisches nicht geupdated.